src/components/replica/index.tsx

```diff
@@ -55,7 +55,7 @@ const ReplicaDuration: FunctionComponent<{ created: Date; ended?: Date }> = ({
   );
 };
 
-const ContainerDuration: FunctionComponent<{ started: Date; ended?: Date }> = ({
+const ContainerDuration: FunctionComponent<{ started: Date; ended: Date }> = ({
```

Comment:
This is still used as possible undefined?
```suggestion
const ContainerDuration: FunctionComponent<{ started: Date; ended?: Date }> = ({
```